src/clarity-angular/modal/modal.ts

```diff
@@ -118,11 +118,13 @@ export class Modal implements OnChanges, OnDestroy {
     @Input("clrModalClosable") closable: boolean = true;
     @Input("clrModalSize") size: string;
     @Input("clrModalStaticBackdrop") staticBackdrop: boolean = false;
-    @Input("clrModalSkipAnimation") skipAnimation: boolean = false;
+    @Input("clrModalSkipAnimation") skipAnimation: string = "false";
```

Comment:
Because the animations are looking for it as a string. When I had it as a boolean, the animations were messed up because it was being passed to the animation/transition as boolean false.

This was the only way I could fix the animation in the modal. It did not affect the wizard, however, because that passed the value down through an input which the animation apparently converted from boolean to string at some point.